Anlab.Mvc/AnlabMvc.csproj

```diff
@@ -55,6 +56,19 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+  <ItemGroup>
+    <Compile Update="Helpers\QueryResource.Designer.cs">

```

Comment:
@jSylvestre usually we put these in a Resources folder, Helpers folder is usually for stuff like dateTime formatting and string extensions